packages/docs/src/pages/guidelines/accessibility.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Accessibility
+---
+
+CMS strives to make its Design System work for as many people as possible, including people with disabilities.
+
+## What we do to help meet and exceed standards
+
+- build accessibility into our [guiding principles](https://github.com/CMSgov/design-system/blob/master/guides/GUIDING-PRINCIPLES.md).
+- include accessibility as part of our design process when working on new or existing components.
+- provide accessibility as a [checkpoint in new components](https://github.com/CMSgov/design-system/blob/master/.github/ISSUE_TEMPLATE/component-validation-checklist.md) created in the design system.
+- add accessibility as a key aspect to much of the [component documentation](https://github.com/CMSgov/design-system/blob/master/guides/WRITING-DOCUMENTATION.md).
+- conduct regular accessibility reviews of new and existing features and documentation for the design system.
+
```

Comment:
We should probably also add our e2e testing that uses axe for automated a11y testing. Also locally we use that browser bookmarklet for a11y as well

Reply:
Hey @bernardwang which browser bookmarklet are you referring to?

Reply:
This one: https://khan.github.io/tota11y/
<img width="302" alt="Screen Shot 2020-03-09 at 10 23 12 AM" src="https://user-images.githubusercontent.com/5424713/76229370-13c5d900-61f0-11ea-89fb-dfe666f247b0.png">

Although in retrospect, I don't think we should mention it. It was legacy tool (we should consider removing it even), and `axe-webdriver` in e2e testing and `axe` browser extension for web testing are much more comprehensive.
Maybe we could even link to Ad Hoc's a11y tool guidance for more info here? https://adhocteam.us/topics/accessibility/index.html

Reply:
I'm adding in a line about using "automated accessibility testing using [axe-webdriverjs](https://github.com/dequelabs/axe-webdriverjs)" but don't think we should link out to the Ad Hoc a11y page